Libraries/WebSocket/RCTSRWebSocket.h

```diff
@@ -61,11 +61,11 @@ extern NSString *const RCTSRHTTPResponseErrorKey;
 
 // Protocols should be an array of strings that turn into Sec-WebSocket-Protocol.
 // options can contain a custom "origin" NSString
-- (instancetype)initWithURLRequest:(NSURLRequest *)request protocols:(NSArray<NSString *> *)protocols options:(NSDictionary<NSString *, NSString *> *)options NS_DESIGNATED_INITIALIZER;
+- (instancetype)initWithURLRequest:(NSURLRequest *)request protocols:(NSArray<NSString *> *)protocols headers:(NSDictionary<NSString *, NSString *> *)headers NS_DESIGNATED_INITIALIZER;
```

Comment:
Yea, these was recently merged via https://github.com/facebook/react-native/commit/9b87e6c860a95fe3d55285314d3d56be06cb7833

I noticed that his original implementation did have an option for using headers but was asked to remove it because only the `origin` header was required by the contributor.